week2salestask/FilterByMonth.js

```diff
@@ -0,0 +1,24 @@
+// Filtering  Sales by Date: Get all sales records for a specific month (e.g., February 2023).
+
+const fs = require('fs');
```

Comment:
While this is ok, you can simply import using require . More straightforwared in this case